nanoCAT/recipes/coordination_number.py

```diff
@@ -0,0 +1,129 @@
+import numpy as np
+from itertools import combinations
+from typing import Dict, Tuple, List
+from scm.plams import Molecule
+from FOX import group_by_values
+from nanoCAT.bde.guess_core_dist import guess_core_core_dist
+
```

Comment:
You may or may have not seen it before, but after importing all function, classes, modules, _etc._ there is generally a constant by the name of ``__all__: List[str] = ...``.

This list is used by Python to help with importing, and it should generally contain the most important function in a module (_i.e._ the ones you actually want to use outside of the module like ``"coordination_number"``)